scripts/test_translation_templates.py

```diff
@@ -0,0 +1,127 @@
+                    not l.startswith(param) for param in line_beginnings_to_ignore
+                ),
+                file.readlines(),
+            )
+        )
+
+
+def print_filename_results(filename, success=True):
+    if success:
+        logger.debug(f"{filename} - NO CHANGES")
+    else:
+        logging.error(f"{filename} - CHANGES FOUND")
+
+
+def build_static_template(output_filepath):
+    subprocess.run(
+        [
+            'pipenv',
+            'run',
+            'pybabel',
+            'extract',
+            '-F',
+            'babel.cfg',
+            '-o',
+            output_filepath,
+            '.',
+        ]
+    )
+
+
+def check_static_template(source_dir, target_dir):
+    source_contents = get_template_content(
+        f"{source_dir}/messages.pot", ignore_context=True
+    )
+    target_contents = get_template_content(
+        f"{target_dir}/messages.pot", ignore_context=True
+    )
+
+    success = source_contents == target_contents
+
+    print_filename_results(f"{source_dir}/messages.pot", success)
+
+    return 0 if success else 1
+
+
+def build_schema_templates(output_dir):
+    docker_client = docker.from_env()
+
+    for schema_name in SCHEMAS_TO_EXTRACT:
+        template_file = f"{schema_name}.pot"
+        schema_file = f"{schema_name}.json"
+
+        docker_client.images.pull('onsdigital/eq-translations:latest')
+        run_results = docker_client.containers.run(
+            'onsdigital/eq-translations',
+            f"pipenv run python -m cli.template_extractor ../eq-survey-runner/data/en/{schema_file} /usr/src/templates",
+            volumes={
+                os.getcwd(): {'bind': '/usr/src/eq-survey-runner', 'mode': 'rw'},
+                output_dir: {'bind': '/usr/src/templates', 'mode': 'rw'},
+            },
+            remove=True,
+        )
+        logger.info(f"Building {output_dir}/{template_file}")
+        logger.info(run_results.decode())
+
+
+def check_schema_templates(source_dir, target_dir):
+    failure_count = 0
+
+    for schema_name in SCHEMAS_TO_EXTRACT:
+        template_file = f"{schema_name}.pot"
+
+        source_contents = get_template_content(f"{source_dir}/{template_file}")
+        target_contents = get_template_content(f"{target_dir}/{template_file}")
+        template_the_same = source_contents == target_contents
+
+        if not template_the_same:
+            failure_count += 1
+
+        print_filename_results(f"app/translations/{template_file}", template_the_same)
+
+    return failure_count
+
+
+if __name__ == '__main__':
+    logger.debug("--- Testing Translations ---")
+
+    with tempfile.TemporaryDirectory(dir='/tmp') as temp_dir:
+        build_static_template(f'{temp_dir}/messages.pot')
+        build_schema_templates(temp_dir)
+
+        static_failures = check_static_template("app/translations", temp_dir)
+        dynamic_failures = check_schema_templates("app/translations", temp_dir)
+
+        if dynamic_failures + static_failures != 0:
```

Comment:
`static_failures` and `dynamic_failures` could be booleans since the count is not used?
```suggestion
        if not all((dynamic_failures, static_failures)):
```